pypeit/core/wavecal/templates.py

```diff
@@ -430,6 +437,64 @@ def xidl_arcspec(xidl_file, slit):
     return wv_vac.value, spec
 
 
+def xidl_hires(xidl_file, specbin=1):
+    """
+    Read an XIDL format solution for Keck/HIRES
+
+    Note:  They used air
+
+    Args:
+        xidl_file (str):
+            Keck/HIRES save file
+
+    Returns:
+
```

Comment:
fill in

Reply:
Actually you wrote this method, so please help out!